### ISSUE TITLE
Conda build recipe for dartsdk

### ISSUE THREAD

**Issue by anctartica** [dart-lang/sdk#33499]
Thank you for taking the time to file an issue!

In order to route, prioritize, and act on this, please include:

* Dart SDK Version (`dart --version`)
* Whether you are using Windows, MacOSX, or Linux (if applicable)
* Whether you are using Chrome, Safari, Firefox, Edge (if applicable)

Missing some or all of the above might make the issue take longer or be
impossible to act on.

----

Is it really an issue? For general questions consider starting with Stack
Overflow:
https://stackoverflow.com/questions/tagged/dart

Also consider our Gitter channel for light-weight/quick discussions:
https://gitter.im/dart-lang/sdk

-----

**Comment by zoechi** [comment]
Can you please add more information what the issue is about?
What is Conda? Can you provide a link?

**Comment by anctartica** [comment]
Yes updated. I accidentally created the issue without entering the description.
